notebooks/266-table-question-answering/266-table-question-answering.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    tqa = pipeline(task="table-question-answering", model=model, tokenizer=tokenizer)
</pre><p>could you please separate result printing and getting result from pipeline (just run pipeline on one line and then print on another line)?</p><p>It is more clear from user perspective.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
done